server/Dockerfile

```diff
@@ -21,16 +30,26 @@ WORKDIR /scalardb
 
 # The path should be relative from build/docker. Running `gradle docker`
 # (provided by com.palantir.docker plugin) will copy this Dockerfile and
-# server.tar and log4j2.properties to build/docker.
+# server.tar, log4j2.properties and database.properties.tmpl to build/docker.
 ADD server.tar .
 
+ENV SCALAR_DB_CONTACT_POINTS 'cassandra'
+ENV SCALAR_DB_CONTACT_PORT '9042'
+ENV SCALAR_DB_USERNAME ''
+ENV SCALAR_DB_PASSWORD ''
+ENV SCALAR_DB_STORAGE 'cassandra'
+ENV SCALAR_DB_TRANSACTION_MANAGER 'consensus-commit'
+ENV SCALAR_DB_ISOLATION_LEVEL ''
+ENV SCALAR_DB_CONSENSUSCOMMIT_SERIALIZABLE_STRATEGY ''
+
 WORKDIR /scalardb/server
 
 COPY log4j2.properties .
+# Support for use cases where environment variables are used to configure the database
```

Comment:
```suggestion
# Support use cases where environment variables are used to configure the database
```

Reply:
I have corrected the problem.
https://github.com/scalar-labs/scalardb/pull/376/commits/b66edbc35965d62fb205e1645d415764db63e9aa